src/CommunicationLayer/CommDeviceControl/InternetConnectionService.cpp

```diff
@@ -25,6 +25,16 @@ InternetConnectionService::InternetConnectionService(
     QObject::connect(this, SIGNAL(setupChannelSignal()), this, SLOT(connectToDataSource()));
     connectionRetryTimer_.setSingleShot(true);
     connect(&connectionRetryTimer_, SIGNAL(timeout()), this, SLOT(connectToDataSource()));
+
+    //Setting up openOpts
+    AmqpClient::Channel::OpenOpts::BasicAuth auth;
+    auth.password = "guest";
+    auth.username = "guest";
+    openOpts_.host = ipAddress_.toStdString();
+    openOpts_.port = port_;
+    openOpts_.auth = auth;
+    openOpts_.vhost = "/";
+    openOpts_.frame_max = 131072;
```

Comment:
Where'd this number come from? I'm assuming this code is a result of the `SimpleAmqpClient` upgrade, just hoping to verify that there's no behavioral change here.

Reply:
There is no behavioral change, that is actually the default number of frame. In the deprecated code it is defaulted to the same value.